DTS-STN/Alpha-Site
# Implement ISR on projects page

# Description

[Implement ISR](https://trello.com/c/AHRiZFGP)

Adding ISR functionality to the projects page, which should invalidate the cache once an hour and fetch new data at that point. Added feature flag `ISR_ENABLED` so that we can easily turn it on or off.

## Acceptance Criteria

Project updates appear on SC Labs without having to run our pipelines.

## Test Instructions

1. Set `ISR_ENABLED`  to true
2. Make sure you're pointing at our dev Strapi api
3. Run the application and navigate to the projects page - make a note of what projects are visible and what state they're in
4. In a separate tab, go to the dev Strapi admin portal
5. Update/add/remove a project
6. Go back to the projects page and notice the change take effect
7. You may need to wait a minute to see the changes

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
